iphone/cli/commands/_build.js

```diff
@@ -1091,13 +1091,13 @@ iOSBuilder.prototype.configOptionPPuuid = function configOptionPPuuid(order) {
 							logger.error(__('Unable to find any non-expired development provisioning profiles that match the app id "%s".', appId) + '\n');
 						}
 						logger.log(__('You will need to log in to %s with your Apple Developer account, then create, download, and install a profile.',
-							'http://appcelerator.com/ios-dev-certs'.cyan) + '\n');
+							'https://developer.apple.com/account/ios/certificate/certificateList.action?type=development'.cyan) + '\n');
```

Comment:
This way redirects for me, but it looks like the new URL is `https://developer.apple.com/account/resources/certificates/list`, I can't find a way to toggle development/production via the URL.

Should we start using the newer URL?

Reply:
Fixed!